src/qml/geometry_editors/FillRingToolBar.qml

```diff
@@ -0,0 +1,107 @@
+import ".."
+import QtQuick 2.12
+import org.qgis 1.0
+import org.qfield 1.0
+import Theme 1.0
+import Utils 1.0
+
+VisibilityFadingRow {
+  id: fillRingToolbar
+
+  signal finished()
+
+  property FeatureModel featureModel
+  readonly property bool blocking: drawPolygonToolbar.isDigitizing
+
+  spacing: 4 * dp
+
+  DigitizingToolbar {
+    id: drawPolygonToolbar
+    showConfirmButton: true
+
+    QuestionDialog{
+      id: questionDialog
+    }
+
+    Loader {
+      id: formPopupLoader
```

Comment:
that's the other thing, using a loader here avoid loading the form on the toolbar loading but later on demand.

Reply:
interesting